src/Tools/dotnet-getdocument/src/Commands/InvokeCommand.cs

```diff
@@ -79,9 +79,16 @@ protected override int Execute()
                             projectName,
                             targetFramework.Version));
                     }
+                    else if (targetFramework.Version > new Version(7, 0))
+                    {
+                        toolsDirectory = Path.Combine(thisPath, "net7.0");
```

Comment:
Should this format from the property value so it doesn't need changing if the TFM gets bumped in a future version?

```suggestion
                        toolsDirectory = Path.Combine(thisPath, $"net{targetFramework.ToString(2)}");
```